lib/ui/pages/account_dashboard.dart

```diff
@@ -0,0 +1,351 @@
+          Container(
+            decoration: const BoxDecoration(
+              color: Colors.white,
+              borderRadius: BorderRadius.only(
+                topLeft: Radius.circular(
+                  20.0,
+                ),
+                topRight: Radius.circular(
+                  20.0,
+                ),
+              ),
+            ),
+            child: Column(
+              children: [
+                Padding(
+                  padding: const EdgeInsets.all(
+                    20.0,
+                  ),
+                  child: Row(
+                    mainAxisAlignment: MainAxisAlignment.center,
+                    crossAxisAlignment: CrossAxisAlignment.center,
+                    children: <Widget>[
+                      Column(
+                        children: <Widget>[
+                          Icon(
+                            icon,
+                            size: 40,
+                          ),
+                          const SizedBox(height: 20),
+                          TitleText(
+                            text: text,
+                            fontSize: 28,
+                            color: textColor,
+                          ),
+                          const SizedBox(height: 20),
+                          Padding(
+                            padding: const EdgeInsets.all(
+                              8.0,
+                            ),
+                            child: ButtonTheme(
+                              minWidth: Get.width * 0.6,
+                              height: 70.0,
+                              child: RaisedButton(
+                                shape: RoundedRectangleBorder(
+                                  borderRadius: BorderRadius.circular(
+                                    18.0,
+                                  ),
+                                  side: BorderSide(
+                                    color: Colors.grey,
+                                  ),
+                                ),
+                                onPressed: () {},
+                                color: LightColor.navyBlue2,
+                                textColor: Colors.white,
+                                child: Text(
+                                  '${t.amount} \$',
+                                  style: const TextStyle(
+                                    fontSize: 28,
+                                  ),
+                                ),
+                              ),
+                            ),
+                          ),
+                        ],
+                      ),
+                    ],
+                  ),
+                ),
+                const Divider(
+                  height: 8,
+                ),
+                Padding(
+                  padding: const EdgeInsets.fromLTRB(
+                    20.0,
+                    0,
+                    20,
+                    0,
```

Comment:
```suggestion
                    0
```
I think better not to use trailing commas for smt like this